images-list

```diff
@@ -673,6 +679,8 @@ quay.io/kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 1.10.7
 quay.io/kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 1.12.2
 quay.io/kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 1.12.3
 quay.io/kiwigrid/k8s-sidecar rancher/mirrored-kiwigrid-k8s-sidecar 1.15.9
+quay.io/minio/minio rancher/minio RELEASE.2022-03-24T00-43-44Z
+quay.io/minio/minio rancher/mirrored-minio RELEASE.2022-03-24T00-43-44Z
```

Comment:
Similar change: rancher/mirrored-minio-minio

Reply:
Will be fixed